features/generator_specs/generator_specs.feature

```diff
@@ -14,3 +14,17 @@ Feature: Generator spec
                 invoke  rspec
                 create    spec/generator/my_generators_generator_spec.rb
           """
+
+    Scenario: Use custom generator with different default path
+        When I run `bundle exec rails generate rspec:install --default-path behaviour --force`
```

Comment:
Features are used for two purposes, to generate the [documentation](https://relishapp.com/rspec/rspec-rails/docs) and for end-to-end testing. So we strive to keep the balance between the two. This scenario seems to be more end-to-end centred.
From the documentation standpoint, it's not clear where the default path setting is persisted.

As far as I see, you've improved the current code in two aspects:
1. It's now able to persist `--default-path` option passed to `rspec:install` into `.rspec`
2. Generators now pick up the default path option from `.rspec`

I suggest separating documentation for those two improvements.
Does it make sense from your standpoint?

If you'd agree, I suggest adding `rspec:install` example as a note to `features/README.md`'s "Configure" section, something like:
```
You can also customize the default spec path with `--default-path` option:

bundle exec rails generate rspec:install --default-path behaviour --force
```
I can't think of a better place this note would fit.

Side note - I really appreciate if you take care of `script/rails generate` along the way, I believe it's a legacy form, and doesn't even work anymore.

For `generator_specs.feature`, instead of running `rspec:install` in a project directory that already has RSpec installed, I'd suggest to directly specify the contents of `.rspec`. You can copy-paste this piece from [this feature](https://github.com/rspec/rspec-core/blob/9de11e52d078696c01c3515351e1e634aada1115/features/configuration/default_path.feature#L25).

And yes, one feature is sufficient. It is a generic document, that was supposed to be a root documentation file for all generators. The table of contents is just broken, and this doc appears separate from other generator docs.

Reply:
Updated!